com.unity.ml-agents/Runtime/Sensors/ISensor.cs

```diff
@@ -92,7 +92,7 @@ public interface ISensor
         /// Return information on the compression type being used. If no compression is used, return
         /// <see cref="CompressionSpec.Default()"/>.
         /// </summary>
-        /// <returns>CompressionSpec used by the sensor.</returns>
+        /// <returns>The type of compression used by the sensor.</returns>
```

Comment:
Maybe 
```suggestion
        /// <returns>An object describing the compression used by the sensor.</returns>
```
(similar to `GetObservationSpec()`)?